src/ios-deploy/ios-deploy.m

```diff
@@ -50,11 +50,18 @@
 const char* lldb_prep_noninteractive_justlaunch_cmds = "\
     run\n\
     safequit\n\
+    kill\n\
+";
+
+const char* lldb_prep_stopapp_cmds = "\
+kill\n\
```

Comment:
Use the same whitespace convention as found in the cmds immediately above and below the change.

Reply:
fixed